README.md

```diff
@@ -140,5 +140,22 @@ AppSpector.run(with: config)
 ```
 <!-- integration-swift-example-end -->
 
+## Start/Stop API
+AppSpector starts automatically.
+To stop AppSpector communicating with web client use start/stop API.  
```

Comment:
+To stop AppSpector from data collection and communication with web client use start/stop API.